app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Roadrunner</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+
+  <!-- required for va_common -->
+ 	<%= split_stylesheet_link_tag "application", :media => "all" %>
+
+	<!--[if lte IE 9]>
+    <link href="/stylesheets/application_split2.css" media="all" rel="stylesheet" type="text/css" />
+	<![endif]-->
+</head>
+<body>
+
+	<!-- Render va header -->
+	<%= render 'va_common/header' %>
```

Comment:
If you look at va_common, main template that does this yield for you. Is that not as good of an idea?